recipe/conda_build_config.yaml

```diff
@@ -1,21 +1,71 @@
-YEAR:
+# corresponding to the clang versions that are live in
+# https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/main/recipe/conda_build_config.yaml
+clang_version:
+  - 14.0.6
```

Comment:
Let's not go back to 14

Reply:
Fine by me, though it's still the default in conda-forge until https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4215 is merged (hence why I kept it).